src/main/java/org/example/Hangman/Hangman.java

```diff
@@ -0,0 +1,94 @@
+package org.example.Hangman;
+
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+public class Hangman {
+    public void start() {
+        Player player1 = new Player();
+        Player player2 = new Player();
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("select first players name");
```

Comment:
Robisz to samo 2 razy a może funkcja do której przekażesz Playera i "first"/"second" 😛

Reply:
Zgadzam się z uwagą. Myślałem żeby skorzystać z wzorca Builder, a to wrzuciłem żeby działało. Czy Builder będzie ok?

Reply:
jak najbardziej ! :D